### PR TITLE
Removed API usages that were deprecated in IntelliJ 2022.1.

### PR DESCRIPTION
**Pull request by odisseus**
This PR is related to #1458.

## Test plan

`./gradlew runPluginVerifier`

### CODE REVIEW

**Comment by odisseus** [comment]
It looks like the plugin verifier tool attempts to download several versions of IntelliJ (which constitute a few GB of data). I have no idea why this step fails in CI, but it might be causedd by exceedding some kind of traffic quota.

File: src/main/java/com/sourcegraph/find/browser/BrowserAndLoadingPanel.java [hunk 1]
```diff
@@ -76,7 +75,7 @@ private void refreshUI() {
 
     } else if (errorMessage != null) {
       String wrappedText = WordUtils.wrap("Error: " + errorMessage, 100);
-      String[] lines = wrappedText.split(SystemUtils.LINE_SEPARATOR);
+      String[] lines = wrappedText.split(System.lineSeparator());
```
Comment (mkondratek) [inline src/main/java/com/sourcegraph/find/browser/BrowserAndLoadingPanel.java]: I remember some troubles with this one. Please check and review the UI in the latest and the oldest supported versions 🙏

Comment (odisseus) [reply src/main/java/com/sourcegraph/find/browser/BrowserAndLoadingPanel.java]: I'm just going to split by `\n`. This should work on all platforms.

Comment (pkukielka) [reply src/main/java/com/sourcegraph/find/browser/BrowserAndLoadingPanel.java]: > I remember some troubles with this one

There was a problem with a separator but it was for UI separator element - maybe you think about that other one?:)

**Review by pkukielka — APPROVED** [review]
LGTM

**Pull request merged**